rust/worker/src/blockstore/arrow/blockfile.rs

```diff
@@ -90,10 +99,43 @@ impl ArrowBlockfileWriter {
     ) -> Result<ArrowBlockfileFlusher, Box<dyn ChromaError>> {
         let mut delta_ids = HashSet::new();
         for delta in self.block_deltas.lock().values() {
+            // Skip empty blocks. Also, remove from sparse index.
```

Comment:
Is there a way we can make it so that these are not present at commit time in the first place? (i.e detect a block is emptied and remove it at build time?)